modules/logging/messages.ts

```diff
@@ -32,16 +34,17 @@ export async function messageDelete(message: Message | PartialMessage): Promise<
 	const content = !shush && messageToText(message, false);
 	const { embeds, files } =
 		shush ? { embeds: [], files: [] } : extractMessageExtremities(message);
-
+	const auditLogs = await config.guild
+		.fetchAuditLogs({ limit: 1, type: AuditLogEvent.MessageDelete })
+		.catch(() => void 0);
 	await log(
 		`${LoggingEmojis.MessageDelete} ${message.partial ? "Unknown message" : "Message"}${
 			message.author ? ` by ${message.author.toString()}` : ""
-		} in ${message.channel.toString()} (ID: ${message.id}) deleted`,
+		} in ${message.url} (Sent: ${time(Math.floor(message.createdTimestamp / 1000))}) deleted ${auditLogs?.entries.first()?.executor ? `by ${auditLogs.entries.first()?.executor?.toString()}` : ""}`,
```

Comment:
use `time(message.createdAt)` so you don't need the math

Reply:
also store the executor in its own variable so you don't need that long thing twice
that'd also fix lint